pkg/nsm/apis/simpledataplane/doc.go

```diff
@@ -1,4 +1,5 @@
-// Copyright (c) 2017 Cisco and/or its affiliates.
```

Comment:
I don't know why GH is showing this as a move of a file from the vendor directory. How bizarre.